DuoSecurity/duo_client_purge.py

```diff
@@ -64,3 +82,13 @@ def get_ldap_user_by_mail(ldap_conn, email):
                   % (duo_email, duo_user_id)
             if NOOP == False:
                 admin_api.update_user(duo_user_id, status='disabled')
+
+    # Delete users that have been disabled and haven't logged in for GRACE_PERIOD seconds (licenses)
+    for duo_entry in all_disabled_duo_usernames:
+        last_login = duo_entry.get('last_login')
+        if last_login:
+            unix_last_login = int(last_login)
+            if int(time.time()) - unix_last_login > GRACE_PERIOD:
```

Comment:
Need to "import time" before this works on python 2.6.6

Reply:
done